### ISSUE TITLE
Support for MQTT retain

### ISSUE THREAD

**Issue by apfelflo89** [technyon/nuki_hub#13]
Could you implement an option for the state topics to configure a retain option? 
This way the MQTT Broker will save the last known state of the lock or battery etc even if the hub is not reachable

**Comment by technyon** [comment]
Good idea, I kind of forgot about it. The retain flag is set in version 4.5
